index.rst

```diff
@@ -27,6 +27,10 @@ The recommended way to install Mink with all its dependencies is through
 
     $ composer require behat/mink
 
+.. note::
+    For local installations of composer you must call it like this: ``$ php composer.phar require behat/mink`` .
```

Comment:
> @aik099 FYI, this CS rule about the wrapping comes from the Symfony documentation standards. I decided to apply the same standards

Thanks god, that you're not considering 85 symbols per line limit from PEAR standard in PHP. Generally I'm against short lines and think that 120 symbols per line is just proper line length limit.